bin/create-tables

```diff
@@ -1,10 +1,33 @@
 #!/usr/bin/env node
 
-// if running locally, you will need to already have created a database
+// Creates a users table and populates it with responses already submitted to Typeform
+// This is to be used when setting up a local instance or environment for the first time
+
+// if running locally, you will first need to create a database
 // Heroku (our current host) creates a database automatically when you
 // install the postgres addon
 
-const createUsersTable = require('../db-helpers').createUsersTable
+const dbHelpers = require('../db-helpers'),
+    typeformHelpers = require('../typeform-helpers'),
+    createUsersTable = dbHelpers.createUsersTable,
+    saveUsers = dbHelpers.saveUsers,
+    getFromTypeform = typeformHelpers.getFromTypeform,
+    parseResponses = typeformHelpers.parseResponses
 
-console.log("creating users table...")
-createUsersTable()
+console.log("Creating users table...")
+createUsersTable( () => {
+    console.log("Getting all past responses from Typeform...")
+    getFromTypeform( {}, (error, response, body) => {
+        if (error) {
+            console.log("Error:", error)
+        } else {
+            console.log("Received data from Typeform.")
+            const users = parseResponses(JSON.parse(body))
+            
+            console.log("Saving past responses to users table...")
+            saveUsers(users, (result) => {
+                console.log(result)
```

Comment:
Instead of relying on an offset, when the database is set up we are going to import all of the past responses into our own database. That way we can always just sort our db by `date_submit` and ask Typeform for the responses after that date.